Description: Remove trailing full-stop in whatis entry
 Trailing full-stop in NAME section causes lexgrog parsing error relating to
 whatis entry
Author: Peter J. Mello <admin@petermello.net>
Origin: Asbru Project Team <https://github.com/asbru-cm/packages/blob/master/libgnome2-gconf-perl/debian/patches/fix-bad-whatis-entry.patch>
Forwarded: no
Last-Update: 2019-05-27

--- a/GConf.pm
+++ b/GConf.pm
@@ -105,7 +105,7 @@ __END__
 
 =head1 NAME
 
-Gnome2::GConf - Perl wrappers for the GConf configuration engine.
+Gnome2::GConf - Perl wrappers for the GConf configuration engine
 
 =head1 SYNOPSIS
 
